### PR TITLE
Update design doc

### PR DESCRIPTION
**Pull request by wangkuiyi**
Fixes https://github.com/k8sp/sextant/issues/202
Fixes https://github.com/k8sp/sextant/issues/247
Fixes https://github.com/k8sp/sextant/issues/258

### CODE REVIEW

**Comment by lipeng-unisound** [comment]
LGTM!

File: doc/sextant-design.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      ```
+      mkdir -p ~/work
+      export GOPATH=$HOME/work
+      ```
+
+   1. 获取Sextant并且获取其中Go程序的依赖
+
+      ```
+      go get https://github.com/k8sp/sextant/...
+      ```
```
Comment (zhengbo0) [inline doc/sextant-design.md]:
```
root@zhengbo:/home/zhengbo# go get https://github.com/k8sp/sextant/...
package https:/github.com/k8sp/sextant/...: "https://" not allowed in import path
```
修改为go get github.com/k8sp/sextant/...

Comment (wangkuiyi) [inline doc/sextant-design.md]: Done

**Comment by pineking** [comment]
LGTM

**Pull request merged**